code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -46,16 +84,17 @@
 	heldname = new_heldname
 	user.log_message("set \"[heldname]\" as a name in a cyborg reclassification board at [loc_name(user)]", LOG_GAME)
 
-/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/R, user = usr)
+/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/borg, mob/living/user = usr)
 	. = ..()
-	if(.)
-		var/oldname = R.real_name
-		var/oldkeyname = key_name(R)
-		R.custom_name = heldname
-		R.updatename()
-		if(oldname == R.real_name)
-			R.notify_ai(AI_NOTIFICATION_CYBORG_RENAMED, oldname, R.real_name)
-		usr.log_message("used a cyborg reclassification board to rename [oldkeyname] to [key_name(R)]", LOG_GAME)
+	if(!.)
+		return .
+	var/oldname = borg.real_name
+	var/oldkeyname = key_name(borg)
+	borg.custom_name = heldname
+	borg.updatename()
+	if(oldname == borg.real_name)
+		borg.notify_ai(AI_NOTIFICATION_CYBORG_RENAMED, oldname, borg.real_name)
+	usr.log_message("used a cyborg reclassification board to rename [oldkeyname] to [key_name(borg)]", LOG_GAME)
```

Comment:
```suggestion
	user.log_message("used a cyborg reclassification board to rename [oldkeyname] to [key_name(borg)]", LOG_GAME)
```

let's use `user` instead here because otherwise the arg is completely useless